### ISSUE TITLE
Replace network graph with nested list

### ISSUE THREAD

**Issue by willzeng** [unitaryfund/metriq-app#347]
The network graph has some drawbacks for displaying a hierarchy:
- It initially looks like this which doesn't show structure
![image](https://user-images.githubusercontent.com/5214594/144073548-4c541186-222a-46f5-bf62-ebdac77ab757.png)
-  Clicking and dragging nodes requires users to manually sort out the structure
- It doesn't scale well to hundreds or thousands of nodes

Replacing this with a nested list (think reddit comments) adds to readability and scalability.

For example something like: https://commonmark.org/help/tutorial/10-nestedLists.html

@WrathfulSpatula @vprusso please do add if there's more from our previous discussion that isn't captured here or if this is a duplicate.

**Comment by nathanshammah** [comment]
Just a comment: Since this come up again in one demo, as the current graph being not helpful, and in case deploying the substitute graph may take some time, I propose to first remove the current visualization, and then add the graph with the nested list or the solution is agreed upon.

**Comment by willzeng** [comment]
+1

**Comment by WrathfulSpatula** [comment]
Agreed. I'll remove the force simulation view right now, before the next demo.

**Comment by WrathfulSpatula** [comment]
This, of all current issues, is particularly important in function for presenting the taxonomy. Let me figure out an implementation.

**Comment by WrathfulSpatula** [comment]
After adding "seed data," I'm about to open a PR that restricts task and method index views to **just top-level items**. From the top level, users can click into child tasks and methods, to drill down to more specific topics.

I looked through freely licensed accordion views, for the purpose of displaying an arbitrary depth tree, and I wasn't impressed with the options, for our purpose. We might add a field that can filter all tasks/methods in the index views, but starting at top-level and clicking through to children works better than an accordion. This is being tracked in #411, and we can close a number of related issues at once, for that PR.

**Comment by willzeng** [comment]
+1 to an eventual accordion view and doing top level in the meantime

**Comment by WrathfulSpatula** [comment]
There are 2 problems I notice with the accordion views that are freely licensed, based on the fact that most of them use indentation to show hierarchy. Firstly, I don't find the tree structure very clear at a glance based just on indentation of the items. Secondly, at arbitrary indentation depth, the views would become a mess, for overflow and line-wrap.

However, now, I'm about to take the UI we liked from top-level categories presented a couple of Fridays ago, apply that to the (newly hierarchical) task and method index views, and add flat hierarchy text filter "searches" to the top of both index views. I think this is faster and easier than multi-level accordion views. However, we'll review at the weekly meeting, and we'll be eager for your critique.

**Comment by WrathfulSpatula** [comment]
@willzeng Given the new top level tasks, which let us click through the child hierarchy successively, and considering the addition of the global text filter on the tasks page, do we really need this as well?

In particular, I didn't like how the tree view packages I found on the NPM registry handled multiple levels of nesting. I worried that our hierarchy would tab right too far, after several nesting levels, and it would hurt aesthetics and make the tree harder to understand. I'm also not sure where this proposed view naturally fits, now that the tasks index page has been updated.
